curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md

```diff
@@ -6,17 +6,17 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-Suppose you have a file that you wish to import all of its contents into the current file. This can be done with the <dfn>import *</dfn> syntax.
+Suppose you have a file and you wish to import all of its contents into the current file. This can be done with the <code>import *</code> syntax.
 Here's an example where the contents of a file named <code>"math_functions"</code> are imported into a file in the same directory:
 <blockquote>import * as myMathModule from "math_functions";<br>myMathModule.add(2,3);<br>myMathModule.subtract(5,3);</blockquote>
 And breaking down that code:
 <blockquote>import * as object_with_name_of_your_choice from "file_path_goes_here"<br>object_with_name_of_your_choice.imported_function</blockquote>
-You may use any name following the <code>import * as </code>portion of the statement. In order to utilize this method, it requires an object that receives the imported values. From here, you will use the dot notation to call your imported values.
+You may use any name following the <code>import * as </code>portion of the statement. In order to utilize this method, it requires an object that receives the imported values (i.e., you must provide a name). From here, you will use the dot notation to call your imported values.
 </section>
 
 ## Instructions
 <section id='instructions'>
-The code below requires the contents of a file, <code>"capitalize_strings"</code>, found in the same directory as it, imported. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
+The code in this file requires the contents of another file, <code>"capitalize_strings"</code>. <code>"capitalize_strings"</code> is in the same directory as the current file. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
```

Comment:
Maybe change : 

> The code in this file requires the contents of another file, <code>"capitalize_strings"</code <code>"capitalize_strings"</code> is in the same directory as the current file.

To :  The code in this file requires the contents of another file, <code>"capitalize_strings"</code <code>"capitalize_strings"</code> that is located in the same directory.

Reply:
Just changed it. Thanks!